ami-recycle-bin/main.tf

```diff
@@ -0,0 +1,144 @@
+provider "aws" {
+ region = "us-west-2"
+}
+
+terraform {
+ required_providers {
+   aws = {
+     source  = "hashicorp/aws"
+     version = "5.65.0"
+   }
+ }
+}
+
+resource "aws_cloudwatch_event_rule" "event_rule" {
+  name                = "invoke-lambda-daily"
+  description         = "Invoke a Lambda function every day"
```

Comment:
```suggestion
  description         = "Invoke a Lambda function once per day"
```

Reply:
> 

Updated `description` to `Invoke a Lambda function once per day`